citrination_client/models/predicted_value.py

```diff
@@ -0,0 +1,32 @@
+class PredictedValue(object):
+    """
+    The value/sigma output from a prediction.
+    """
+
+    def __init__(self, key, value, sigma=None):
+        """
+        Constructor.
+
+        :param key: The descriptor key for the prediction
+        :param value: The predicted value
+        :param sigma: If the predicted value is a real number, the sigma
+            for the prediction
+        """
+        self._key = key
+        self._value = value
+        if type(value) == str:
+            self._sigma = None
+        else:
+            self._sigma = sigma
```

Comment:
put this in the setter!

Reply:
actually remove it completely

Reply:
(if you call it loss)

Reply:
ask julia/erin
a) do we want to call it standard dev or loss
b) should we include it for categorical predictions

Reply:
erin thinks `loss` is fine and to include it for both result types

Reply:
👍